util/gh-pages/index.html

```diff
@@ -94,6 +94,24 @@
         @media (min-width: 992px) {
             .search-control {
                 margin-top: 0;
+                float: right;
+            }
+        }
+
+        @media (min-width: 400px) {
+            #upper-filters {
+                display: flex;
+            }
+        }
+
+        @media (max-width: 412px) {
+            #version-filter-selector {
+                right: 0;
+                left: auto;
+            }
+
+            #version-filter-count {
+                display: none;
             }
```

Comment:
I was testing how it looked on mobile through Firefox dev tools, and that was the width of the first device I saw. I don't know how accurate it is, I just really didn't like seeing the filter count go out of bounds 😅.

Reply:
I don't know any mobile phone that has such a small screen size. Therefore, I would drop this filter. In the worst case it'll wrap sooner :)

Reply:
As it happens my phone (Nexus 5X) has a CSS width of 412px 😄

CSS pixels are multiplied by the [device pixel ratio](https://developer.mozilla.org/en-US/docs/Web/API/Window/devicePixelRatio), so for phones the number will be smaller than you'd expect

Reply:
Ahh, I forgot about that, thanks for pointing that out. In that case, I would still prefer the filters to wrap earlier rather than removing the count indicator. Especially since the other counters stay. Would that change be alright with you?

Reply:
This is the `2` in the screenshot above? Yeah sounds good to me for consistency

Reply:
Yeah, that's the `2` the other filters have a similar indicator :)